build.gradle

```diff
@@ -33,9 +33,14 @@ allprojects {
         }
 
         //Provides all Adobe AEM specific repos (like cq-workflow-console)
+        //May no longer be needed since latest jar files are not on the public Adobe repo
         maven {
             url "http://repo.adobe.com/nexus/content/groups/public"
         }
+
+        // hack to allow getting to a working version of AEM 6.1 workflow-console jar
+        // without running afoul of licensing restrictions
+        flatDir { dirs '.' }
```

Comment:
Rather than making people put it in the subdirectory, use the top of the project directory:
```
flatDir { dirs rootProject.projectDir }
```